server/src/main/java/com/google/sps/utils/Ad.java

```diff
@@ -0,0 +1,225 @@
+/* 
+ * Author: Kira Toal
+ * Date: June 24, 2020
+ */
+package com.google.sps.utils;
+import java.util.List; 
+ 
+public class Ad {
+  // fields
+  public String id; 
+  public String advertiser;  
+  public String startDate; // isostring format
+  public String endDate; //isostring
+  public long impressionsMin; 
+  public long impressionsMax; 
+  public boolean isTargetingAge; 
+  public List<String> genderTargets; 
+  public List<String> geoTargets;
+  public long spendMin; 
+  public long spendMax; 
+  public String headline; 
+  public String link;
+  public String content;  
+  public String headlineSentiment; 
+  public String headlineTerms;
+  public String contentSentiment; 
+  public String contentTerms; 
+  // setter methods
+  public void setId(String id) {
+    this.id = id; 
+  }
+  public void setAdvertiser(String advertiser) {
+    this.advertiser = advertiser; 
+  }
+  public void setStartDate(String startDate) {
+    this.startDate = startDate; 
+  }
+  public void setEndDate(String endDate) {
+    this.endDate = endDate; 
+  }
+  public void setImpressionsMin(long impressionsMin) {
+    this.impressionsMin = impressionsMin; 
+  }
+  public void setImpressionsMax(long impressionsMax) {
+    this.impressionsMax = impressionsMax; 
+  }
+  public void setIsTargetingAge(boolean isTargetingAge) {
+    this.isTargetingAge = isTargetingAge; 
+  }
+  public void setGenderTargets(List<String> genderTarget) {
+    this.genderTargets = genderTargets; 
+  }
+  public void setGeoTargets(List<String> geoTargets) {
+    this.geoTargets = geoTargets;
+  }
+  public void setSpendMin(long spendMin) {
+    this.spendMin = spendMin; 
+  }
+  public void setSpendMax(long spendMax) {
+    this.spendMax = spendMax; 
+  }
+  public void setHeadline(String headline) {
+    this.headline = headline; 
+  }
+  public void setLink(String link) {
+    this.link = link; 
+  }
+  public void setContent(String content) {
+    this.content = content; 
+  }
+  public void setHeadlineSentiment(String headlineSentiment) {
```

Comment:
For a future PR, consider making a set of Objects to encapsulate some of this information so it's not a whole bunch of isolated fields.  It seems like there is a logical grouping of some of these fields (like text, sentiment, terms -- an object representing this set of fields could be applied to both headline and content).

Reply:
Ok